tst/ogb/model_and_data_utils.py

```diff
@@ -96,4 +124,39 @@ def add_zeros(data: Data) -> Data:
     :return:
     """
     data.x = torch.zeros(data.num_nodes, dtype=torch.long)
-    return data
\ No newline at end of file
+    return data
+
+i=0
+def to_line_graph(data: Data, directed: bool = True) -> Data:
+    """
+    Convert a graph G to its corresponding line-graph L(G)
+    :param data: a torch_gemoetric Data object representing representing a graph
+    """
+    global i
+    print(i)
```

Comment:
Are those lines still necessary or are they debug statements that can be safely removed?